src/libs/common.go

```diff
@@ -222,7 +223,7 @@ func CheckCommandLineConfig() {
 		os.Exit(0)
 	}
 
-	viper.SetConfigName(GetEnv("CONF_FILE_NAME", "conf"))
+	viper.SetConfigName(GetEnv("CONF_FILE_NAME", "local"))
```

Comment:
Lets not change this if it is not impacting ... there might be some auto-test frameworks that might break.